### DISCUSSION TITLE
State of development

### DISCUSSION THREAD

**Discussion by PulellaP** [oxen-io/lokinet#1787] [Q&A] [answered]
I have been looking around on the web as to what the state of development is for lokinet. Is it still in alpha? Beta? full release?

**Comment by majestrate** [comment, marked as answer]
> I have been looking around on the web as to what the state of development is for lokinet. Is it still in alpha? Beta? full release?

lokinet works and is stable but there isn't any absolute compatibility guarantees currently defined. as for if this constitutes alpha/beta/lambda/sigma or whatever greek letter release I have no idea as those have no concrete definition. 
i guess it's 'fully released' but i don't even know what that really means either.
